build_resolvers/lib/src/resolver.dart

```diff
@@ -272,30 +269,6 @@ class AssetBasedSource extends Source {
 
   @override
   bool get isInSystemLibrary => false;
-
-  Source resolveRelative(Uri relativeUri) {
```

Comment:
public API? breaking change?

Reply:
we don't export this class, so no it's not breaking

Reply:
`AnalyzerResolvers` is the only class that's public from this file
https://github.com/dart-lang/build/blob/c49f2a6330624fc86ea470d5b2ad141b03ed9cea/build_resolvers/lib/build_resolvers.dart#L5